src/app/Agent/AgentLiveProbes.tsx

```diff
@@ -320,7 +320,7 @@ export const AgentLiveProbes: React.FC<AgentLiveProbesProps> = (_) => {
     return (
       <ErrorView
         title={'Error retrieving active probes'}
-        message={`${errorMessage}. Note: This is commonly caused by the agent not being loaded/active, check that the target was started with the agent (-javaagent:/path/to/agent.jar).`}
+        message={`${errorMessage}.  Note: This error is generally caused when the agent is not loaded or is not active. Check that the target was started with the agent (-javaagent:/path/to/agent.jar).`}
```

Comment:
There is an extra space before `Note:`. Same for `src/app/Agent/AgentProbeTemplates.tsx`.